tests/algorithms/pytorch/test_base_algo.py

```diff
@@ -283,7 +284,11 @@ def test_rng_state_save_and_load(network, train_linear_nodes, session_dir, rng_s
     expected_output_round_2 = torch.rand(n_rng_sample)
 
     algo_deps = Dependency(
-        pypi_dependencies=["torch", "numpy"],
+        pypi_dependencies=[
+            "torch==2.0.1",
+            "numpy==1.24.3",
+            "--extra-index-url https://download.pytorch.org/whl/cpu",
+        ],
```

Comment:
Wouldn't it be nice to use a fixture or at least a constant for this?

Reply:
Yep you're right !

Reply:
Nice, thank you!